src/main/java/com/rpietraszewski/medicalclinic/validator/PatientValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.rpietraszewski.medicalclinic.validator;
+
+import com.rpietraszewski.medicalclinic.exception.PatientIdCardNoChangeException;
+import com.rpietraszewski.medicalclinic.exception.PatientNullFieldsException;
+import com.rpietraszewski.medicalclinic.model.dto.PatientCreateDTO;
+import com.rpietraszewski.medicalclinic.model.entity.Patient;
+
+import static com.rpietraszewski.medicalclinic.MedicalApplicationHelper.patientHasNullFields;
+
+public class PatientValidator {
```

Comment:
1. nie chcemy zeby taka klasa byla rozszerzana -> powinnismy zrobic ze to jest public final class
2. nie chcemy aby mozliwe bylo utworzenie w ogole instancji takiej klasy -> nie ma to sensu bo sa same metody statyczne -> w tym celu powinnismy zdefiniowac prywatny konstruktor bezargumentowy mozemy to zrobic przy uzyciu lomoboka nad klasa:
@NoArgsConstructor(access = AccessLevel.PRIVATE)